scripts/licenses/licenses/git.go

```diff
@@ -81,5 +81,5 @@ func gitRemoteURL(repoPath string, remoteName string) (*url.URL, error) {
 		}
 		return u, nil
 	}
-	return nil, fmt.Errorf("Git remote %q does not have a valid URL", remoteName)
+	return nil, fmt.Errorf("git remote %q does not have a valid URL", remoteName)
```

Comment:
nit: Strictly speaking, this should be capitalized, since Git is a proper noun.

Reply:
Reworded.